### ISSUE TITLE
I can't see my own reblogs on my dashboard

### ISSUE THREAD

**Issue by mb7823** [new-xkit/XKit#1329]
I can't see stuff I've reblogged or that people reblog from me on my dashboard and I am not sure why. I have checked all the extensions and nothing has that feature. Please help?

**Comment by greyfiel** [comment]
This is in the extension called _Tweaks_. Scroll down to the section called _Dashboard Tweaks_ and there's a checkbox labelled _Don't show my own posts on my dashboard_. Underneath it, there's another one labelled _Don't show liked posts on my dashboard_. Are you sure you don't have either of these checked?

**Comment by mb7823** [comment]
Yes, I am sure, I don't have that extension installed.  Thank you though.

**Comment by greyfiel** [comment]
What extensions _do_ you have installed? That would help.

**Comment by mb7823** [comment]
Activity+, Drafts+, Enhanced Queue, Editable Reblogs, Messaging Tweaks, Mass+, Notifications+, No Recommended, One-Click Postage, One-Click Reply, Read Posts, Reblog Yourself, Search Likes, Shorten Posts, Tag Tracking+, View My Tags, XInbox, Anti Capitalism, Post Block, and Quick Tags.

**Comment by nightpool** [comment]
Does this still happen after disabling New XKit? If not, can you post your Extension Info Export, from XKit > Other > Export Extension Information?

**Comment by mb7823** [comment]
https://gist.github.com/b2798d33cd8aef122a0225527d537dcb

**Comment by nightpool** [comment]
So it looks like you do in fact have the tweaks extension installed. Can you try disabling that to see if it resolves the issue?

Otherwise, can you post a picture of your chrome://extensions page? That'll help me in narrowing this down further.
